driver/csiplugin/connectors/rest_v2.go

```diff
@@ -709,7 +709,10 @@ func (s *SpectrumRestV2) ListCSIIndependentFilesets(ctx context.Context, filesys
 	klog.V(4).Infof("[%s] rest_v2 ListCSIIndependentFilesets. filesystem: %s", utils.GetLoggerId(ctx), filesystemName)
 
 	encodedFilesetComment := strings.ReplaceAll(FilesetComment, " ", "%20")
-	getFilesetURL := fmt.Sprintf("scalemgmt/v2/filesystems/%s/filesets?fields=filesetName&filter=config.parentId=0,config.comment=%s", filesystemName, encodedFilesetComment)
+	url := fmt.Sprintf("scalemgmt/v2/filesystems/%s/filesets?fields=filesetName", filesystemName)
+	filter := fmt.Sprintf("filter=config.isInodeSpaceOwner=true,config.comment=%s", encodedFilesetComment)
```

Comment:
Also to filter independent filesets, we should filter on parentId=0.

Reply:
The dependent filesets can also have parentId=0 if they use the inode space of root fileset, so we can't use this for filter